servers.controllers.js

```diff
@@ -55,3 +56,19 @@ exports.getComments = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.postComments = (req, res, next) => {
+  const { username, body } = req.body;
+
+  const { article_id } = req.params;
+
+  checkIfArticleExists(article_id)
```

Comment:
This is unnecessary as if the article doesn't exist a PSQL error with be thrown as the article_id is foreign key on the comments table.